zboxcore/sdk/writemarker_mutex.go

```diff
@@ -363,6 +363,7 @@ func (wmMu *WriteMarkerMutex) lockBlobber(
 		if !shouldContinue {
 			break
 		}
+		WMLockWaitTime *= 2 //In each round we increase the wait time by factor 2.
```

Comment:
Instead of fix sleep time / wait time, we are increasing wait time
by factor 2 and starting with low wait time. When there are multiple
goroutines (say n)  trying to lock Blobber, then we no longer need to wait
O(n * wait_time) time.

Reply:
Lets not make WMLockWaitTime a variable instead make it constant.
Waittime is increasing at some condition and never decreasing.
For simplicity lets put:
`const WMLockWaitTime = time.Second * 5`
To wait simply multiply it by some factor but never update the value